### PR TITLE
Fix: fix home and alert , badge , button , avtar and image code

### PR DESCRIPTION
**Pull request by Divya70**
Refactor Landing page 
Installation page 
Alert
Badge
button 
avtar
image

### CODE REVIEW

File: docs.css [hunk 1]
```diff
@@ -1,195 +1,217 @@
...
+
+::-webkit-scrollbar {
+      width: 10px;
+
+}
+
+::-webkit-scrollbar-thumb {
+      background-color: var(--grey-color);
+      border-radius: 30px;
+
```
Comment (Nishant-96) [inline docs.css:25]: Suggestion Try to use relative units for above 10 px values.

File: pages/Badge/badge.css [hunk 2]
```diff
@@ -1,255 +1,209 @@
...
+  margin-left: 2rem;
+}
+
+.badge {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 50px;
+  height: 50px;
```
Comment (ScoobyDubeyDo) [inline pages/Badge/badge.css:21]: Try using Rems or any other responsive unit

File: docs.css [hunk 3]
```diff
@@ -1,195 +1,217 @@
...
+.sidebar-title {
       margin-top: 7rem;
+      color: var(--black-color);
 }
 
-aside{
-    display: flex; 
+aside {
+      display: flex;
```
Comment (Nishant-96) [inline docs.css:97]: Suggestion Avoid element grabing as user might get unwanted style..

File: pages/Badge/badge.css [hunk 4]
```diff
@@ -1,255 +1,209 @@
...
-    height: 460px;
-    border:0; 
-    margin-left: 2rem; 
-   }
+.badge-on-avtar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 10px;
+  height: 200px;
```
Comment (ScoobyDubeyDo) [inline pages/Badge/badge.css:138]: Use rems

File: docs.css [hunk 5]
```diff
@@ -1,195 +1,217 @@
...
+      color: var(--black-color);
       font-size: 2rem;
       padding: 10px;
-     
+
 }
-p{
+
+.installation-des {
       width: 900px;
```
Comment (Nishant-96) [inline docs.css:155]: Suggestion Try to use relative units

File: pages/Badge/badge.css [hunk 6]
```diff
@@ -1,255 +1,209 @@
...
+  height: 15px;
+  bottom: 30px;
+  right: 25px;
+}
+
+.badge-img3 {
+  width: 10px;
+  height: 10px;
+  bottom: 55px;
+  right: 51px;
```
Comment (ScoobyDubeyDo) [inline pages/Badge/badge.css:207]: Don't use px

File: pages/Alert/alert.css [hunk 7]
```diff
@@ -1,43 +1,11 @@
...
-      padding: 10px;
-      color: var(--white-color);
-      font-size: 1.1rem;
-      width: 900px;
-      padding-left: 5rem;
-}
-.alerts-div{
+.alerts-div {
       border: 1px solid rgb(192, 192, 192);
       height: 550px;
```
Comment (Nishant-96) [inline pages/Alert/alert.css:3]: Suggestion Try to use relative units for above 10 px values.

File: pages/Avtar/avtar.css [hunk 8]
```diff
@@ -1,124 +1,85 @@
...
-      font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
-      color: var(--white-color);
+      font-family: Cambria, Cochin, 'Times New Roman';
+      color: var(--secondary-color);
 }
-.avtar-text1{ 
+
+.avtar-text1 {
       width: 200px;
       height: 200px;
```
Comment (Nishant-96) [inline pages/Avtar/avtar.css:65]: Suggestion Try to use relative units.

File: pages/Avtar/avtar.css [hunk 9]
```diff
@@ -1,124 +1,85 @@
...
-      font-size: 3rem; 
+      font-size: 3rem;
 }
-.avtar-text2{
+
+.avtar-text2 {
       width: 150px;
       height: 150px;
       font-size: 2.4rem;
```
Comment (Nishant-96) [inline pages/Avtar/avtar.css:72]: Suggestion Try to use relative units for above 10 px values.

File: pages/Form/form.css [hunk 10]
```diff
@@ -1,87 +1,96 @@
...
-     border: 2px solid red;
-     border-radius: 0.4rem;
+
+#password {
+      border: 2px solid red;
+      border-radius: 0.4rem;
 }
-#pass{
+
+#pass {
```
Comment (ScoobyDubeyDo) [inline pages/Form/form.css:56]: Try using classes in place of ids

File: pages/Badge/badge.css (hunk 4 above)
Comment (Nishant-96) [inline pages/Badge/badge.css:138]: Suggestion Try to use relative units for above 10 px values.

File: pages/Badge/badge.css [hunk 11]
```diff
@@ -1,255 +1,209 @@
...
+  margin-top: 10px;
+  height: 200px;
+  width: 90%;
+  border: 1px solid var(--container-border);
+  margin-left: 2rem;
+}
+
+.badge-image1 img {
+  height: 300px;
+  clip-path: circle();
```
Comment (Nishant-96) [inline pages/Badge/badge.css:146]: Suggestion Try to use relative units for above 10 px values.

**Review by ScoobyDubeyDo — APPROVED** [review]

File: pages/documentation.css [hunk 12]
```diff
@@ -133,79 +204,122 @@ aside li a:hover {
...
+    font-size: 1rem;
+    width: 900px;
+    padding-left: 5rem;
+}
+
+.text-point {
+    padding: 10px;
+    color: var(--white-color);
+    font-size: 1rem;
+    width: 900px;
```
Comment (Nishant-96) [inline pages/documentation.css:271]: Suggestion Try to use relative units

**Review by Nishant-96 — APPROVED** [review]

**Pull request merged**